presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/hadoop-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+fail() {
+  echo "$(basename "$0"): $*" >&2
+  exit 1
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+echo "[$(date)] $0: configuring hadoop services"
+
+$DIR/apply-site-xml-override.sh /etc/hive/conf/hive-site.xml "$DIR/hive-site-overrides.xml" || fail "Could not apply hive-site-overrides.xml"
```

Comment:
Please quote all variables in bash (https://google.github.io/styleguide/shell.xml#Variable_expansion):

```suggestion
"${DIR}/apply-site-xml-override.sh" /etc/hive/conf/hive-site.xml "${DIR}/hive-site-overrides.xml" || fail "Could not apply hive-site-overrides.xml"
```

nit: you don't need to `fail` explicitly, because you run with `set -e`, so bash will exit with error upon any command that exits with non-0 exit code